src/test/kotlin/no/nav/soknad/arkivering/dto/DtoObjectMother.kt

```diff
@@ -1,26 +1,27 @@
 package no.nav.soknad.arkivering.dto
 
 import org.joda.time.DateTime
+import org.junit.jupiter.api.Test
 
 
-	//forsendelse informasjon
+//forsendelse informasjon
 	private val innsendingsidIdForBilForsendelse = "IS123456"
 	private var erEttersendelseBil = false
 	private val personIDBil = "12345678910"
 	private val temaBil = "BIL"
```

Comment:
Tror ikke det er lurt å lage tester som tilsynelatende er knyttet til en spesifikk søknadstype da SoknadsMottaker er en generisk tjeneste som ikke vil ha spesialhåndtering for de enkelte søknadstypene eller temaene. Hvis BIL søknad en gang i framtiden tas bort, vil det skape usikkerhet i forbindelse med opprydding.

Reply:
tanken er å bygge opp object mother med forskjellige scenarioer basert på de søknadene vi skal faktisk flytte på. Bil er bare det første "søknadsobjektet". siden lager vi flere forskjellige og ungår tulledata, men at vi faktisk har mer ekte data. det har vært tanken.